browser/js/topics/topic.html

```diff
@@ -46,83 +53,80 @@
-                      {{ numVotes.resource[resource.id] || 0 }}
-                    </button>
-                    <button id="btn-resource-voted-{{ resource.id }}" type="button" class="btn btn-xs btn-primary hidden" ng-click="devote('resource', resource.id)">
-                      <span class="glyphicon glyphicon-thumbs-up"></span>
-                      {{ numVotes.resource[resource.id] || 0 }}
-                    </button>
-                    <!-- Add a tag -->
-                    <button id="btn-add-tag-{{ resource.id }}" type="button" class="btn btn-xs btn-default" ng-click="addTag(resource.id)" ng-if="isLoggedIn">
-                      <span class="glyphicon glyphicon-tags"></span>
-                      &nbsp;add tag
-                    </button>
-                    <!-- Add to plan -->
-                    <button ng-click="addToPlan(resource.id)" type="button" class="btn btn-xs btn-default" ng-if="isLoggedIn">
-                      <span class="glyphicon glyphicon-plus"></span>
-                      {{selectedPlan.name}}
-                    </button>
-                    <!-- Flag this resource -->
-                    <button type="button" class="btn btn-xs btn-default" ng-if="isLoggedIn" ng-click="addFlag(resource.id)">
-                      <span class="glyphicon glyphicon-flag"></span>
-                    </button>
+            <div ng-repeat="resource in resources | filter:searchText">
+              <div class="clearfix" ng-if="$index % 3 == 0"></div> <!-- fixes column<->overflow -->
+              <div class="col-md-4">
+                <div class="panel panel-default resource-panel">
+                  <div class="panel-heading">
+                    <div class="resource-title">
+                      <h3 class="panel-title"><a href="{{ resource.url }}" target="_blank">{{ resource.name }}</a>&nbsp;</h3>
+
+                      <!-- Glyphicon represents resource type -->
+
+                    </div>
+                    <div class="resource-action-box">
+                      <!-- Resource type -->
+                      <span class="btn btn-xs btn-default">
+                        <span class="glyphicon glyphicon-pencil" ng-if="resource.type === 'tutorial'"></span>
+                        <span class="glyphicon glyphicon-film" ng-if="resource.type === 'video'"></span>
+                        <span class="glyphicon glyphicon-text-height" ng-if="resource.type === 'article'"></span>
+                        <span class="glyphicon glyphicon-book" ng-if="resource.type === 'book'"></span>
+                        <span class="glyphicon glyphicon-th-list" ng-if="resource.type === 'documentation'"></span>
+                        {{ resource.type }}
+                      </span>
+                      <!-- Voting -->
+                      <button id="btn-resource-upvote-{{ resource.id }}" type="button" class="btn btn-xs btn-default" ng-click="upvote('resource', resource.id)">
```

Comment:
They're used in a jQuery fn to show/hide the upvote/voted button.  We'll look to shift the resource boxes to a directive.